lib/presentation/breed_layout.dart

```diff
@@ -1,61 +1,37 @@
-import 'package:mocked_list_of_breeds/business_logic/cubit/breed_list_cubit.dart';
-import 'package:mocked_list_of_breeds/business_logic/cubit/breed_state.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:mocked_list_of_breeds/business_logic/riverpod/data_provider.dart';
+import 'package:mocked_list_of_breeds/data/model/breed.dart';
 import 'package:mocked_list_of_breeds/widgets/breed_widget.dart';
-import 'package:mocked_list_of_breeds/widgets/breed_widget_error.dart';
 
-class BreedLayout extends StatefulWidget {
+class BreedLayout extends ConsumerWidget {
   const BreedLayout({super.key});
 
   @override
-  State<BreedLayout> createState() => _BreedLayoutState();
-}
-
-class _BreedLayoutState extends State<BreedLayout> {
-  final nameOfScreen = "Dogs List Screen";
-
-  final double heightOption = 60;
-
-  final double thicknessOption = 1;
-
-  @override
-  void initState() {
-    context.read<DogListCubit>().fetchBreedApi();
-    super.initState();
-  }
-
-  @override
-  Widget build(BuildContext context) {
-    return MaterialApp(
-      home: Scaffold(
-        appBar: AppBar(
-          title: Text(nameOfScreen),
-          backgroundColor: Colors.black,
-        ),
-        body: BlocBuilder<DogListCubit, BreedState>(
-          builder: (context, state) {
-            if (state is InitialState) {
-              return const Center(child: Text("Waiting"));
-            } else if (state is LoadingState) {
-              return const Center(child: CircularProgressIndicator());
-            } else if (state is ErrorState) {
-              return BreedError(
-                  textError: 'Something wrong with $state in DogListScreen');
-            } else if (state is LoadedState) {
-              return ListView.separated(
-                  itemBuilder: (_, index) =>
-                      BreedWidget(state.breedList[index]),
-                  separatorBuilder: (_, __) => const Divider(
-                        thickness: 4,
-                      ),
-                  itemCount: (state.breedList.length));
-            } else {
-              throw Exception('unprocessed state $state in DogListLayout');
-            }
-          },
-        ),
+  Widget build(BuildContext context, WidgetRef ref) {
+    final _data = ref.watch(breedsDataProvider);
+    return Scaffold(
+      appBar: AppBar(
+        backgroundColor: Colors.black,
+        title: const Text("Breed List"),
       ),
+      body: _data.when(
+          data: (_data) {
+            List<Breed> breedList = _data.map((e) => e).toList();
+            return ListView.separated(
+              itemCount: breedList.length,
+              itemBuilder: (_, index) => BreedWidget(breedList[index]),
+              separatorBuilder: (_, __) => const Divider(
+                color: Colors.grey,
+                height: 60,
+                thickness: 1,
+              ),
+            );
+          },
+          error: (err, s) => Text(err.toString()),
```

Comment:
no ess and s, please, give normal names for argument, and move all data like 60 to const